MainTemplate.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE HTML>
 <html><head>
-  <title>Main Template Page</title> <!-- Page title. This appears in the browser tab and status bar. -->
+  <title>WebTV Unavailable</title> <!-- Page title. This appears in the browser tab and status bar. -->
```

Comment:
Why are you changing the template pages to "Maintenance pages"? Why not just create a new page?